lib/build_targets/package.dart

```diff
@@ -129,7 +129,44 @@ class DotnetTpk extends TizenPackage {
       copyDirectory(pluginsLibDir, libDir);
     }
 
-    // Run the .NET build.
+    assert(tizenSdk != null);
+    // The output TPK is signed with an active profile unless otherwise
+    // specified.
+    String? securityProfile = buildInfo.securityProfile;
+    final SecurityProfiles? securityProfiles = tizenSdk!.securityProfiles;
+    if (securityProfile != null) {
+      if (securityProfiles == null ||
+          !securityProfiles.contains(securityProfile)) {
+        throwToolExit('The profile $securityProfile does not exist.');
+      }
+    }
+    if (securityProfile == null && securityProfiles != null) {
+      securityProfile = securityProfiles.active;
+    }
+    if (securityProfile != null) {
+      environment.logger
+          .printStatus('The $securityProfile profile is used for signing.');
+    } else {
+      throwToolExit(
+        'No certificate profile found. You can create one using the Certificate Manager.\n'
+        'https://github.com/flutter-tizen/flutter-tizen/blob/master/doc/install-tizen-sdk.md#create-a-tizen-certificate',
+      );
+    }
+
+    File locateTpk(String stdout) {
+      // "Runner -> /path/to/project/tizen/bin/Debug/tizen40/com.example.app-1.0.0.tpk"
```

Comment:
The path depending on the app type and build type (debug, release). I think this is an unnecessary comment.
or `/path/to/project/tizen/bin/{App and build type}/tizen40/com.example.app-1.0.0.tpk"` ?

Reply:
This just represents the output pattern for the regular expression in the next line. Here's an example of the MSBuild output.

```sh
$ dotnet build tizen/service/
MSBuild version 17.3.2+561848881 for .NET
  Determining projects to restore...
  All projects are up-to-date for restore.
  Tizen.Flutter.Embedding -> /home/swift/Git/flutter-tizen/embedding/csharp/Tizen.Flutter.Embedding/bin/Debug/tizen40/Tizen.Flutter.Embedding.dll
  RunnerService -> /home/swift/Git/dummy/csharp_multi_app/tizen/service/bin/Debug/tizen40/RunnerService.dll
  Configuration :
  Platform :
  TargetFramework :
  TizenTpkFiles : tizen-manifest.xml
  RunnerService is signed with Default Certificates!
  RunnerService -> /home/swift/Git/dummy/csharp_multi_app/tizen/service/bin/Debug/tizen40/com.example.csharp_multi_app-1.0.0.tpk

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:02.14
```

We only parse the part `RunnerService -> ...` from this output. `RunnerService`, `Debug`, `tizen40`, `com.example.csharp_multi_app-1.0.0` are all subject to change and do not matter.